### ISSUE TITLE
Don't highlight titles of read threads

### ISSUE THREAD

**Issue by ichthyosaurus** [szopin/harbour-sfos-forum-viewer#43]
I can't use the SDK at the moment, but I think this should do it:

```
in FirstPage.qml in the Label item with 'text: title' @ 284:
old:
                            color: highlighted || item.hasNews ? Theme.highlightColor
                                                               : Theme.primaryColor

new:
                            color: highlighted || item.hasNews
                                   ? Theme.highlightColor
                                   : (item.lastPostNumber < highest_post_number
                                      ? Theme.primaryColor
                                      : Theme.secondaryColor)
```

**Comment by szopin** [comment]
Works great, thanks!

**Comment by szopin** [comment]
Fixed in 0.9.3
